source/Projectile.cpp

```diff
@@ -283,17 +285,21 @@ void Projectile::Move(vector<Visual> &visuals, vector<Projectile> &projectiles)
 
 
 // This projectile hit something. Create the explosion, if any. This also
-// marks the projectile as needing deletion.
+// marks the projectile as needing deletion if it has run out of hits.
 void Projectile::Explode(vector<Visual> &visuals, double intersection, Point hitVelocity)
 {
-	clip = intersection;
 	distanceTraveled += dV.Length() * intersection;
```

Comment:
I may be misunderstanding but cant we just reset the distance once we're done with a target? (Or like just distanceTraveled = ...)

If that doesnt work I'd just have said dropoff and penetration on a laser are not compatible given we dont even intend to use it?
Caus if you want to combine it I dont see how it makes sense that distance affects the projectile but going trough a ship doesnt buy anyway...

Reply:
I've changed it so that Projectile::GetInfo adds the intersection range to the distance traveled when telling the DamageProfile how far the projectile went for damage dropoff calculations. This way the projectile's distanceTraveled doesn't need to be updated and the projectile doesn't need an extra variable.